Client/Frontend/Browser/Tabs/TabTrayViewModel.swift

```diff
@@ -52,6 +52,7 @@ class TabTrayViewModel {
 
     let profile: Profile
     let tabManager: TabManager
+    let overlayManager: OverlayModeManager
```

Comment:
@OrlaM bringing to your attention I added the `OverlayManager` in `TabTrayViewModel` to trigger openNewTab state when the + button is pressed on iPhone. I tried to call this function on BVC delegates chain of the action but it was causing side effects where the keyboard was raising when it shouldn't